training/distributed_training/pytorch/data_parallel/roberta/roberta.ipynb

```diff
@@ -0,0 +1,418 @@
+{
```

Comment:
<p>We don't support MXNet. Please remove MXNet from the intro sentence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3420/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>